src/recipes/tooling-custom-resources.md

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Custom Resources
+
+Custom `Resource`s are exposed for the end user to use within their development, but cannot provide any functionality automatically.
+For example, you can create a custom `AudioStream` type that handles a new and interesting audio file type.
+
+
+## Registering a Resource
+
+This workflow is incredibly similar to the [Hello World Example](/intro/hello-world.md)
+
+```rust
+#[derive(GodotClass)]
+#[class(tool, init, base=Resource)] // use tool!
+struct ResourceType {
+    #[base]
+    base: Base<Resource>,
+}
+
+#[godot_api]
+impl IResource for ResourceType {}
+
+#[godot_api]
+impl ResourceType {
+    #[func]
+    fn foo(&mut self) {
+        godot_print!("bar");
+    }
+}
+```
+
+Notably, the above resource does not export any variables. While not all `Resource`s will require exported variables, most do.
+You can read more about the many options you have for exporting variables
+[in the API docs](https://godot-rust.github.io/docs/gdext/master/godot/prelude/derive.GodotClass.html#properties-and-exports).
```

Comment:
Now that I wrote #26, you could maybe link to the properties page instead of the API docs.

I'll merge that PR soon, so it should work directly.